README.md

```diff
@@ -1,2 +1,15 @@
 # .github
-GitHub metadata and shared workflows
+
+GitHub metadata and shared workflows. The repo name `.github` has special significance to GitHub within an organization, so don't treat this as a normal code repository.
+
+## Shared Workflows
+
+GitHub requires shared workflows to be in a public repo, we are storing them in here. These workflows are specific to Lumio and aren't believed to be generally useful outside our organziation, but they are MIT licensed.
```

Comment:
I guess unless it's enterprise, but effectively for now, it does

Reply:
self-hosted enterprise yeah (not enterprise cloud)